src/utils/dbc.js

```diff
@@ -1,4 +1,4 @@
-const dbcDict = require('./dbc_dict.json');
+const dbcDict = require('./dbc.json');
```

Comment:
maybe I should call this dbc map or something? dbc dict is confusing with the type in openpilot